modules/ting_search/tests/ting_search.test

```diff
@@ -15,8 +15,8 @@ class TingSearchTestCase extends DrupalWebTestCase {
 
     parent::setUp('ting', 'ding_entity', 'search', 'ting_search', 'nanosoap');
     variable_set('ting_agency', '100200');
-    variable_set('ting_search_url', 'http://opensearch.addi.dk/next_2.0/');
-    variable_set('ting_recommendation_url', 'http://openadhl.addi.dk/1.1/');
+    variable_set('opensearch_url', 'http://opensearch.addi.dk/next_2.0/');
```

Comment:
As the other place test should be updated as 2.0 do not exists anymore.

Reply:
:+1: